pkgs/development/mobile/androidenv/ndk-bundle/default.nix

```diff
@@ -11,22 +11,37 @@ deployAndroidPackage rec {
   inherit package os;
   nativeBuildInputs = [ makeWrapper ]
     ++ lib.optionals stdenv.isLinux [ autoPatchelfHook ];
-  autoPatchelfIgnoreMissingDeps = true;
-  buildInputs = lib.optionals (os == "linux") [ pkgs.zlib ];
+  buildInputs = lib.optionals (os == "linux") [
+    pkgs.stdenv.cc.cc.lib
+    pkgs.zlib
+
+    # Python module dependencies.
+    pkgs.bzip2
+    pkgs.libxcrypt-legacy
+    pkgs.ncurses5
+  ];
 
   patchElfBnaries = ''
+
     # Patch the executables of the toolchains, but not the libraries -- they are needed for crosscompiling
-    if [ -d $out/libexec/android-sdk/ndk-bundle/toolchains/renderscript/prebuilt/linux-x86_64/lib64 ]; then
-      addAutoPatchelfSearchPath $out/libexec/android-sdk/ndk-bundle/toolchains/renderscript/prebuilt/linux-x86_64/lib64
+    if [ -d $out/libexec/android-sdk/ndk-bundle/toolchains/renderscript/prebuilt/linux-x86_64/lib ]; then
+      addAutoPatchelfSearchPath $out/libexec/android-sdk/ndk-bundle/toolchains/renderscript/prebuilt/linux-x86_64/lib
     fi
 
-    if [ -d $out/libexec/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/lib64 ]; then
-      addAutoPatchelfSearchPath $out/libexec/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/lib64
+    if [ -d $out/libexec/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/lib ]; then
+      addAutoPatchelfSearchPath $out/libexec/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/lib
     fi
 
-    find toolchains -type d -name bin -or -name lib64 | while read dir; do
-      autoPatchelf "$dir"
-    done
+    if [ -d $out/libexec/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/python3/lib ]; then
+      addAutoPatchelfSearchPath $out/libexec/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/python3/lib
+    fi
+
+    # find toolchains -type d -name bin -or -name lib | while read dir; do
+    #   autoPatchelf "$dir"
+    # done
```

Comment:
I should remove instead of just commenting out, this `find` loop, but I am worried I don't understand something here, because `toolchains/llvm/prebuilt/linux-x86_64` is the only thing in `toolchains` AFAICT, and so shouldn't `autoPatchelf toolchains/llvm/prebuilt/linux-x86_64` and autoPatchelf toolchains` be equivalent?

And then the `find` *filters down* the recursion `autoPatchelf` will do. So I don't understand why the old loop I commented out would try to patch Android `.so`s, but `autoPatchelf toolchains/llvm/prebuilt/linux-x86_64` is fine?